components/account/AccountList.tsx

```diff
@@ -28,7 +28,7 @@ import BigNumber from 'bignumber.js'
 interface Column {
   id: 'rank' | 'address' | 'balance' | 'percentage' | 'txsCount';
   label: string;
-  align: string;
+  align: any;
```

Comment:
This is being used in the line below

https://github.com/forbole/big-dipper-celo-client/blob/01c339c0f1e50d290473a7d1bfd735f131777ae4/components/account/AccountList.tsx#L149

I think we can directly use the `TabelCellProps` interface or the `align`.

https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/TableCell/TableCell.d.ts#L15